src/wizards/services/WizardService.ts

```diff
@@ -165,4 +184,10 @@ export class WizardService {
     const wizardState = this.getWizardState(wizardManager);
     return wizardState.stages[wizardState.indexOfActive];
   }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  private deleteVotesForPartneredDeal(wizardManager: WizardManager) {
```

Comment:
I would like that we avoid "deal" specific stuff in  the WizardService.

Reply:
[refac(submit): move congrats modal logic to submitStage](https://github.com/PrimeDAO/prime-deals-dapp/pull/227/commits/eca5c530dd3edb10647f03890b0bed27b2bd13f8)